### PR TITLE
Deploy to new server

### PR DESCRIPTION
**Pull request by erikarvstedt**

### CODE REVIEW

**Comment by nixbitcoin** [comment]
[Fixup branch](https://github.com/nixbitcoin/nixbitcoin.org/commits/deployment-nb)

**Comment by erikarvstedt** [comment]
This PR is currently deployed and ready to merge. The unsquashed version is [here](https://github.com/erikarvstedt/nixbitcoin.org/commits/deployment-unsquashed).

File: backup.nix [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        "var/lib/machines"
+        "var/lib/private"
+        "var/lib/systemd"
+      ];
+
+      repo = "nixbitcoin@freak.seedhost.eu:borg-backup";
+      doInit = false;
+      encryption = {
+        mode = "repokey";
+        passCommand = "cat ${secretsDir}/backup-encryption-password";
```
Comment (nixbitcoin) [inline backup.nix:91]: I think we can safely delete `${secretsDir}/backup-encryption-env` now that duplicity is no longer in use.

File: maintenance/maintenance.sh [hunk 2]
```diff
@@ -59,3 +59,54 @@ xdg-open http://localhost:10000
...
+# Update the postgresql datadir from schema 11.1 to 14 (2022-07-02)
+ssh nixbitcoin.org 'du -sh --apparent /var/lib/postgresql/11.1' # => 47G
+# Duration:
+# real 0m10.088s
+
+ssh nixbitcoin.org 'ls -al /var/lib/postgresql'
+
+# Undo migration (requires that postgresql was not run with the new datadir.)
+ssh nixbitcoin.org 'mv /var/lib/postgresql/<schema>/global/pg_control{.old,}'
+>>>>>>> 83548c4 (maintenance: add postgresql schema update guide)
```
Comment (nixbitcoin) [inline maintenance/maintenance.sh]: This seems to be a leftover?

**Pull request merged**